fastly/block_fastly_service_v1_gzip.go

```diff
@@ -173,6 +175,33 @@ func (h *GzipServiceAttributeHandler) Read(d *schema.ResourceData, s *gofastly.S
 
 	gl := flattenGzips(gzipsList)
 
+	// Note: Although "content_types" and "extensions" fields are optional in spec,
```

Comment:
For consistency in go projects I typically uppercase these types of comments like `// NOTE: ...`